From 80ef8402736fd24b94f616ed51358b066cc1ec81 Mon Sep 17 00:00:00 2001
Message-Id: <80ef8402736fd24b94f616ed51358b066cc1ec81.1297440280.git.jdenemar@redhat.com>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Thu, 10 Feb 2011 12:38:24 +0100
Subject: [PATCH] Fix typo in parsing of spice 'auth' data

https://bugzilla.redhat.com/show_bug.cgi?id=676374

A typo s/spice/vnc/ caused parsing of the spice 'auth' data
to write into the wrong part of the struct, blowing away
other unrelated data.

* src/conf/domain_conf.c: s/vnc/spice/ in parsing spice auth
(cherry picked from commit 10713b1b98268a6eb05ba0912c1f4643c25f2564)
---
 src/conf/domain_conf.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/conf/domain_conf.c b/src/conf/domain_conf.c
index 05fa0cd..9d65855 100644
--- a/src/conf/domain_conf.c
+++ b/src/conf/domain_conf.c
@@ -3872,7 +3872,7 @@ virDomainGraphicsDefParseXML(xmlNodePtr node, int flags) {
 
         def->data.spice.listenAddr = virXMLPropString(node, "listen");
         def->data.spice.keymap = virXMLPropString(node, "keymap");
-        if (virDomainGraphicsAuthDefParseXML(node, &def->data.vnc.auth) < 0)
+        if (virDomainGraphicsAuthDefParseXML(node, &def->data.spice.auth) < 0)
             goto error;
 
         cur = node->children;
-- 
1.7.4

